### ISSUE TITLE
Questions regarding license and features

### ISSUE THREAD

**Issue by schmittl** [hjanetzek/alfons#8]
Hi,
just found out about this project, it seems very valuable and I have some questions. 

I cannot find information about the license, am I correctly assuming that it is also under the MIT-license like https://github.com/tangrams/tangram-es ?

Which of the internationalization features does alfons aim to provide / or not provide?

- Font selection and fallback mechanism
- Script itemization
- Bidi
- Complex text shaping
- Unicode line breaking
- Glyph metrics information
- Text Layout like justification
- Maybe caching strategies for words
- etc.

This information would probably be useful to have in the README as well.
Thanks!

**Comment by hjanetzek** [comment]
Hi! - sorry for the late response. I've added a 2-clause BSD license for the whole project as it matches the Chronotext sources that we used. 

The current development depends on what we need for tangram-es. 

TextShaper class contains most of the logic for Bidi/Script itemization and feeding this into harfbuzz for shaping.There is currently no word shaping cache - but we plan to implement it at some point.
